cfme/tests/containers/test_provider_crud.py

```diff
@@ -42,5 +42,5 @@ def test_provider_edit_port(request, a_container_provider):
     old_port = a_container_provider.port
     with update(a_container_provider):
         a_container_provider.port = '1234'
-    request.addfinalizer(a_container_provider.update({'port': old_port}))
+        request.addfinalizer(lambda: a_container_provider.update({'port': old_port}))
```

Comment:
I think that this request should go straight after ```old_port```, but I agree that the result of the callable is not the right way to go, but as we are fixing it, let's move it to where it should be.